### PR TITLE
Render breadcrumbs dynamically

### PR DESCRIPTION
**Pull request by JonathanHallam**
This removes the hardcoded breadcrumbs now that the `find-coronavirus-local-restrictions` page has a content item. There are some other code changes that are in place either to allow the tests to pass or to ensure that the landing page can be accessed without the hardcoded breadcrumbs.

The only visual difference is the addition of the footer. The breadcrumbs and the sidebar will remain the same
#### Computer
Screenshot 2020-11-12 at 12 01 39

#### Phone
Screenshot 2020-11-12 at 12 01 46

#### Computer
Screenshot 2020-11-12 at 12 00 43

#### Phone
Screenshot 2020-11-12 at 12 01 02

### CODE REVIEW

**Review by leenagupte — COMMENTED** [review]
Looks good to me! 🎉 

I haven't approved this yet, because there's still the lingering question of whether we need to  add the contextual sidebar as well.

**Comment by benthorner** [comment]
Removed CD warning.

We're disabling automatic deployments for this app until it meets the safety criteria identified by RFC 128.

https://github.com/alphagov/collections/pull/2059

**Review by hannahjump — COMMENTED** [review]
Footer looks good from my perspective

File: app/views/coronavirus_local_restrictions/show.html.erb [hunk 1]
```diff
@@ -49,7 +49,7 @@
   </div>
   <div class="govuk-grid-column-one-third">
     <div class="coronavirus-local-restriction__related-navigation">
-      <%= render 'govuk_publishing_components/components/related_navigation', {
+      <%= render 'govuk_publishing_components/components/contextual_sidebar', {
```
Comment (leenagupte) [inline app/views/coronavirus_local_restrictions/show.html.erb:52]: Could we tag the related content in content-tagger and pass in the whole content instead, like we do for the breadcrumb?

For example: https://github.com/alphagov/government-frontend/blob/960e79d99c05f48cb110d007b11ad31e7d802970/app/views/shared/_sidebar_navigation.html.erb#L6

**Review by leenagupte — COMMENTED** [review]
Looks really good! 🎉 
I just have lingering question about the sidebar.

File: app/views/coronavirus_local_restrictions/show.html.erb (hunk 1 above)
Comment (JonathanHallam) [reply app/views/coronavirus_local_restrictions/show.html.erb:52]: I did try this and it wouldn't render anything. I wonder if the `parsed_content` thing in your example is doing something special. I'll have a look :)

Comment (JonathanHallam) [reply app/views/coronavirus_local_restrictions/show.html.erb]: Sorted now that it's been properly tagged in content tagger

**Review by leenagupte — APPROVED** [review]
Looks good! 👍 💯 
Yay for deleting snowflake config! 🥇

**Pull request merged**